api/v1beta1/openstackdataplaneservice_types.go

```diff
@@ -76,6 +76,17 @@ type OpenStackDataPlaneServiceSpec struct {
 	// OpenStackAnsibleEERunnerImage image to use as the ansibleEE runner image
 	// +kubebuilder:validation:Optional
 	OpenStackAnsibleEERunnerImage string `json:"openStackAnsibleEERunnerImage,omitempty" yaml:"openStackAnsibleEERunnerImage,omitempty"`
+
+	// AnsibleMaxFailPercentage is used to tune service specific, allowable failure percentages during the Ansible execution
+	// https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_error_handling.html#setting-a-maximum-failure-percentage
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:validation:Maximum=100
+	AnsibleMaxFailPercentage int `json:"ansibleMaxFailPercentage,omitempty"`
+
+	// AnsibleAnyErrorsFatal is used to tune service specific, any_errors_fatal
+	// +kubebuilder:validation:Optional
+	AnsibleAnyErrorsFatal bool `json:"ansibleAnyErrorsFatal,omitempty"`
```

Comment:
Having a bool with omitempty means the False value will always be ignored. Therefore this field once set to True cannot be explicitly set to False, as when the client sends False the json serializer after the webhook code will omit the False value from the json body as False is the empty value. You probably need a *bool to allow differentiation between False value and case when value is not provided.

Reply:
Ah nice. Thanks!